binaries/scheduler/main.go

```diff
@@ -17,21 +19,22 @@ func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
 	flag.Parse()
 
-	parser := config.DefaultParser()
-	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
-	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
-	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	env, o := config.Default()
```

Comment:
When it comes to config I very much value Explicit easy to understand definitions, even if it requires more lines of code.  You want the operational values of a system to be easy to understand.  

Every time I've seen a clever config system it generally becomes fairly unmaintainable.  I did a fairly large rewrite of the Cuckoo one on Observability to make it easy to test and understand.  It meant writing more lines of config and code, however this code is easy to write and test.  Its super simple.  So low cost to create and maintain